nautobot/ipam/views.py

```diff
@@ -818,6 +848,14 @@ def post(self, request):
             },
         )
 
+    def assign_ipaddresses_to_interface(self, pks, request):
+        ip_addresses = IPAddress.objects.filter(pk__in=pks)
+        interface_model = Interface if "interface" in request.GET else VMInterface
+        interface_id = request.GET.get("interface") or request.GET.get("vminterface")
+        interface = interface_model.objects.get(id=interface_id)
```

Comment:
Another place that is almost identical to the blocks in 714-717 and 766-769. Can we please try to use this same code in all places?